pkg/expression/bench_test.go

```diff
@@ -1672,6 +1672,12 @@ func testVectorizedBuiltinFunc(t *testing.T, vecExprCases vecExprBenchCases) {
 				t.Fatalf("evalType=%v is not supported", testCase.retEvalType)
 			}
 
+			if _, ok := baseFunc.(*builtinAddSubDateAsStringSig); ok {
```

Comment:
In previous code before this PR, the testing context will ignore the invalid date errors without appending warnings and cover up the bug: https://github.com/pingcap/tidb/issues/50197 . In this PR, we have to skip the below check because #50197 fixed.